src/styles/popup-information.css

```diff
@@ -21,7 +21,7 @@
   background-color: var(--information-box-background-color);
   box-shadow: 0 1px 4px rgba(0, 0, 0, 0.2);
   padding: 15px;
-  border-radius: 4px;
+  border-radius: var(--boc-border-radius);
```

Comment:
`box-border-radius`, no?

Reply:
yes I change the them value and all occurences